src/components/Search.js

```diff
@@ -0,0 +1,97 @@
+import React, { Fragment } from 'react';
+import { Global, css } from '@emotion/core';
+import { DocSearch } from '@docsearch/react';
+import '@docsearch/react/style';
```

Comment:
Does this work when you run `yarn build`?

I thought Gatsby needed `gatsby-browser.js` for this ([see example](https://github.com/francoischalifour/reactjs.org/blob/656358b1bd1b41dcd212b0db2ca285e1c3e02600/gatsby-browser.js#L16)).

Reply:
This is required indeed. Fixing that. You can now see the output of the build command (`/public/`) available from https://popper-docsearch-v3.surge.sh/docs/v2/